### ISSUE TITLE
The configuration File resets after every server launch

### ISSUE THREAD

**Issue by Syguyful** [DoggyCraft/MyDog#11]
I keep trying to adjust the numbers of attack damage and health per level, however every time I launch the server any changes I made to the command file are reverted back to default values. Is there any way to prevent this from happening? Thanks for your reply!

**Comment by Fido2603** [comment]
I cannot reproduce this issue on my end.. Are you making the changes while the server is running, and if so have you done a `/md reload` after the config changes, as to save the config changes?
I will close the issue for now, please reopen the issue with some reproduction steps! :)
And thank you very much for creating the issue!

**Comment by Fido2603** [comment]
From #12: "so if you could provide me with version of MyDog, server version, and maybe try to put the plugin on Debug, and then please send the the startup logs (use something like Pastebin or Gist)."

**Comment by Fido2603** [comment]
Closing because of inactivity and the lack of decent reproduction steps.

**Comment by JT033** [comment]
Hi, sorry for reacting to a closed issue nut I'm having the same problem and I haven't been able to pin point the cause. I noticed it happened today after I used /reload. I tried /MyDog reload, but it didn't fix it. Also restarted/closed the server but the wolf reverted to it's level 1 value (Health: 20.0 Damage: 4.0) and kept his XP.

I'm using PaperMC 1.15.2
The MyDog version is v0.1.0
Pastebin of startup: [https://pastebin.com/7diUVR8N](https://pastebin.com/7diUVR8N)

This happens a lot and I'm not really sure where it's coming from, the debug doesn't show any errors.

**Comment by Fido2603** [comment]
Hi,

I'll take a look into this once I get some time, as I am currently very booked up with work ^^
Reopening...

Cheers,
Hero

**Comment by LordGriefington** [comment]
same here, after the restart  the health is at 20 again. I was trying it without the other plugins to be sure they doesn't conflict, JT033 was using paper too, so I was trying the normal spigot. but its always the same. the rest works fine and I am on 1.18! good job (I use default configs). maybe it helps if you work on it again.

**Comment by HeroGamers** [comment]
@LordGriefington thanks for using the plugin! And glad to hear that it works alright on 1.18 as well (though I bet the teleporting needs some finetuning).

There's quite a lot of stuff that needs to be updated though, and I haven't had the time nor motivation to work on plugins for quite a while now, with having started at university, etc.
I am however planning on coming back to work on MyDog and other plugins, when the community that I am an administrator at goes on an updating spree.

Might give this bug a shot here in early January, and update some stuff to 1.18 as well while I'm at it, but don't take my word for it haha.

Cheers, and thanks again for using the plugin! Means a lot and helps with the motivation to get back to working on it <3

**Comment by HeroGamers** [comment]
Looks like the issue happened after Spigot changed the way that entities are loaded.
Before I checked for entities in the chunk, whenever a chunk was loaded, but somehow the list of entities was empty.

Added a new event, [EntitiesLoadEvent](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/event/world/EntitiesLoadEvent.html), which should fix this issue.

Currently fixed on 8f8865f, and will be on v0.1.2.
